README.md

```
@@ -1,3 +1,75 @@
+ㅁ 포 ㅁ ㅁ ㅁ ㅁ ㅁ 포 ㅁ
+ㅁ ㅁ ㅁ ㅁ 초 ㅁ ㅁ ㅁ ㅁ
+차 상 마 사 ㅁ 사 상 마 차
+```
+
+## 1.2 기물 이동
+
+### 이동 입력
+
+(움직이기 전 위치) (기물이름) (움직일 위치)
+
+ex) 03 마 84
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)